### DISCUSSION TITLE
Telegram login

### DISCUSSION THREAD

**Discussion by chris6611** [lncrawl/lightnovel-crawler#1197] [Ideas]
Hello, is it possible to add some form of /login command to the Telegram bot where we can enter a username and password to log into a novel site before we paste the link so that we can do different login credentials per session rather than having to restart the whole instance of the bot with new --login credentials?

Or rather, if I am misunderstanding and the bot does not support logins at all, is it possible to add this functionality to it?

**Comment by dipu-bd** [comment]
Currently the bots do not support login. The feature is only available with console app
